### PR TITLE
a few cleanups for client output

### PR DESCRIPTION
**Pull request by hqhq**
Signed-off-by: Qiang Huang h.huangqiang@huawei.com

### CODE REVIEW

File: api/client/pause.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ func (cli *DockerCli) CmdPause(args ...string) error {
 	for _, name := range cmd.Args() {
 		if _, _, err := readBody(cli.call("POST", fmt.Sprintf("/containers/%s/pause", name), nil, nil)); err != nil {
 			fmt.Fprintf(cli.err, "%s\n", err)
-			encounteredError = fmt.Errorf("Error: failed to pause container named %s", name)
+			encounteredError = fmt.Errorf("Error: failed to pause one or more containers")
```
Comment (LK4D4) [inline api/client/pause.go]: Can we replace such stuff with slice of errNames and return error with all of them at the end if it's not nil?

Comment (tiborvass) [inline api/client/pause.go]: let's just add the names of all the containers that failed. More info can be found on the daemon.

**Comment by hqhq** [comment]
@LK4D4 @tiborvass Updated.

**Comment by runcom** [comment]
cool! encounteredError always looked ugly to me :smile: thanks!

**Comment by icecrime** [comment]
Code LGTM

**Comment by jfrazelle** [comment]
LGTM

**Pull request merged**